### PR TITLE
disable ability to retrieve permission and messages for multiple businesses or customers

### PR DESCRIPTION
**Pull request by defra-dom**

### CODE REVIEW

File: test/integration/graphql/schema.test.js [hunk 1]
```diff
@@ -673,6 +724,21 @@ describe('schema', () => {
             permissionGroups: [CustomerBusinessPermissionGroup]
           }
 
+          """Represents a business owned by a customer."""
+          type CustomerBusinesses {
```
Comment (rkdefra) [inline test/integration/graphql/schema.test.js:728]: Plurality

File: app/graphql/types/business/business.gql [hunk 2]
```diff
@@ -88,6 +88,36 @@ type BusinessType {
   type: String
 }
 
+"""
+Represents the a customer of a business.
+"""
+type BusinessCustomers {
```
[the remark this answers is not in the document]
Comment (defra-dom) [reply app/graphql/types/business/business.gql:94]: `BusinessCustomer` is already taken for a single BusinessCustomer tho?

File: app/graphql/types/customer/customer.gql [hunk 3]
```diff
@@ -189,6 +189,31 @@ type CustomerBusiness {
   permissionGroups: [CustomerBusinessPermissionGroup] @on
 }
 
+"""
+Represents a business owned by a customer.
+"""
+type CustomerBusinesses {
```
[the remark this answers is not in the document]
Comment (defra-dom) [reply app/graphql/types/customer/customer.gql:195]: same as before

File: app/graphql/types/business/business.gql (hunk 2 above)
[the remark this answers is not in the document]
Comment (rkdefra) [reply app/graphql/types/business/business.gql:94]: Ok, so just update the comment above to reflect the semantics of the type:
```
"""
Represents the a customers of a business.
"""

File: app/graphql/types/business/business.gql [hunk 4]
```diff
@@ -112,6 +112,37 @@ type BusinessCustomer {
...
 
+  """
+  The role of the customer against the business.
+  """
+  role: String @on
+}
+
+"""
+Represents the customer of a business. Additional properties over BusinessCustomer:
+ - permissionGroups
```
Comment (rkdefra) [inline app/graphql/types/business/business.gql:123]: :+1: :+1:

**Review by rkdefra — APPROVED** [review]
Much clearer, thanks @defra-dom !

**Pull request merged**